--- cproto-4.6/configure.in.orig	Sat Jan 24 02:42:05 1998
+++ cproto-4.6/configure.in	Sat Oct 20 16:33:07 2001
@@ -57,19 +57,17 @@
 AC_DEFINE_UNQUOTED(CPP,"$CPP")
 
 ###	output makefile and config.h
-changequote({,})dnl
 AC_OUTPUT(Makefile config_h,
-{
+[
 echo creating config.h
 sed	-e '/^# /d' \
-	-e 's/ -D/\
+	-e 's/ \?-D/\
 #define /g' \
-	-e 's/\(#define [A-Za-z_][A-Za-z0-9_]*\)=/\1	/g' \
+	-e ['s/\(#define [A-Za-z_][A-Za-z0-9_]*\)=/\1	/g'] \
 	-e 's/\\//g' \
 	config_h >config.h
-}
+]
 echo removing config_h
 rm config_h
 )
-changequote([,])dnl
 touch stamp-h
